This patch removes mention of Debian package manager and binary checks,
as those will be handled by pacman itself. Additionally, wget is made to use
form of progress reporting that plays bit better with terminals.

diff --git a/kinect_fetch_fw b/kinect_fetch_fw
index 3459dca..91b1289 100755
--- a/kinect_fetch_fw
+++ b/kinect_fetch_fw
@@ -9,12 +9,6 @@
 # and/or modify it under the terms of the Do What The Fuck You Want
 # To Public License, Version 2, as published by Sam Hocevar. See
 # http://sam.zoy.org/wtfpl/COPYING for more details.
-#
-# wget and 7z from p7zip-full are needed, they can be installed with
-#   sudo aptitude install wget p7zip-full
-#
-# NOTE: p7zip-full >= 9.20 is required in order to extract .msi files
-#       correctly
 
 set -e
 
@@ -24,9 +18,6 @@ SDK_MD5="40764fe9e00911bda5095e5be777e311"
 [ $# -lt 1 ] && { echo "usage: $(basename "$0") <firmware destdir>" 1>&2; exit 1; }
 FW_DESTDIR=$(readlink -m $1)
 
-command -v wget >/dev/null 2>&1 || { echo "$(basename "$0"): command 'wget' is needed." 1>&2 ; exit 1; }
-command -v 7z >/dev/null 2>&1 || { echo "$(basename "$0"): command '7z' is needed." 1>&2; exit 1; }
-
 TEMPDIR=$(mktemp -d)
 trap 'rm -rf "$TEMPDIR" >/dev/null 2>&1' 0
 trap "exit 2" 1 2 3 15
@@ -44,7 +35,7 @@ EOM
 
 cd "$TEMPDIR"
 ARCHIVE_NAME=$(basename "$SDK_URL")
-rm -f "$ARCHIVE_NAME" && wget "$SDK_URL" -O "$ARCHIVE_NAME"
+rm -f "$ARCHIVE_NAME" && wget --progress=dot:mega "$SDK_URL" -O "$ARCHIVE_NAME"
 
 ARCHIVE_MD5=$(md5sum "$ARCHIVE_NAME" | grep --only-matching -m 1 '^[0-9a-f]*')
 if [ "$ARCHIVE_MD5" != "$SDK_MD5" ];
